src/storage/src/monitor/monitored_storage_metrics.rs

```diff
@@ -93,7 +93,9 @@ impl MonitoredStorageMetrics {
         buckets.push(16.0); // 16s
 
         // 1ms - 100s
-        let state_store_read_time_buckets = exponential_buckets(0.001, 10.0, 5).unwrap();
+        let mut state_store_read_time_buckets = exponential_buckets(0.001, 10.0, 5).unwrap();
```

Comment:
Can you helo modify the comment accordingly? I think the range is 1ms - 80s

Reply:
because 10s is not enough for slow query ...